src/components/sort-menu/SortMenu.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Box, MenuItem } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import Select from '@mui/material/Select'
+import { styles } from '~/components/sort-menu/SortMenu.styles.js'
+import AppButton from '~/components/app-button/AppButton'
+
+const SortMenu = ({ handleChange }) => {
+  const { t } = useTranslation()
+  const [sort, setSort] = React.useState('newest')
```

Comment:
It would be better to put this value into searchParams. Such way will enable sort value to persist between page reloads

Reply:
I tried to implement this.

Reply:
Thank you!